src/node_zlib.cc

```diff
@@ -607,9 +607,11 @@ class CompressionStream : public AsyncWrap, public ThreadPoolWork {
     return AllocForBrotli(data, real_size);
   }
 
+  static constexpr size_t reserveSizeAndAlign =
```

Comment:
```suggestion
  static constexpr size_t kReserveSizeAndAlign =
```
Not 100% sure but I think this doesn't fit the code style, it seems inconsistent already in the codebase
Quite some places use a `k` prefix, others use define style (e.g. `RESERVER_SIZE_AND_ALIGN`).

Reply:
that sounds like pure evil, are you suggestion to make it a #define in a c++ project?

Reply:
Sorry, I didn't mean to convert it to a define, `constexpr` is perfectly fine and the way to go.

It's just about nitpicking about the name. Unfortunatelly [cpp-style-guide](https://github.com/nodejs/node/blob/main/doc/contributing/cpp-style-guide.md) doesn't clarify this.

Looking into code I found several places using the `k` prefix (e.g. [here](https://github.com/nodejs/node/blob/f692878dec6354c0a82241f224906981861bc840/src/node_snapshotable.cc#L1515)), and some using `ALL_UPPER_CASE` (e.g. [here](https://github.com/nodejs/node/blob/f692878dec6354c0a82241f224906981861bc840/src/node_zlib.cc#L153-L154)).

Personally I prefer the proposed `k` prefix.

Reply:
Understood, I guess I can rename it to reserveSizeAndAlign_, although rather ugly, that seems to comply with the code standard / general implementations. I doubt I have ever seen a k though, unless in kilo. Why would you prefer a k there?

Reply:
I don't know where the `k` prefix comes from but it is used at quite some places in this repo for constants. Therefore I see it as helpful as it jumps into my mind that this is a constant. Would I go for `k` prefix in my personal repo: no - but thats a different topic.
The casing you use now is used nowhere in this repo for constants.

Aynhow, all I'm asking is to try to follow the existing style8s). I guess `kReserveSizeAndAlign`, `RESERVE_SIZE_AND_ALIGN`, `reserver_size_and_align` (and maybe more) follow exisiting styles.